core/src/main/java/org/ehcache/core/spi/store/tiering/AuthoritativeTier.java

```diff
@@ -64,6 +64,33 @@
   boolean flush(K key, ValueHolder<V> valueHolder);
 
   /**
+   * Injects the {@link InvalidationValve} to be used by this {@code AuthoritativeTier}.
+   * <P>
+   *   The role of the invalidation valve is to provide a way for the {@code AuthoritativeTier} to force invalidation of
+   *   {@link CachingTier} entries when that is required.
+   * </P>
+   *
+   * @param valve the valve to use for triggering invalidations
+   */
+  void setInvalidationValve(InvalidationValve<K> valve);
+
+  /**
+   * Invalidation valve, that is the mechanism through which an {@link AuthoritativeTier} can request invalidations
+   * from the {@link CachingTier}.
+   *
+   * @param <K> the key type
+   */
+  interface InvalidationValve<K> {
+
+    /**
+     * Requests an invalidation of all {@link CachingTier} mappings.
```

Comment:
Is there a reason this is "all mappings"?  Would "some" mappings suffice?  Do we call this thing in some kind of loop or not?